### PR TITLE
HWPV-36: Speed up closed claims in extract

### PR DESCRIPTION
**Pull request by psoleckimoj**

### CODE REVIEW

File: app/services/data/get-closed-claim-status.js [hunk 1]
```diff
@@ -20,14 +20,6 @@ module.exports = function (claimId) {
...
-      if (event) {
-        if (event !== null && event !== undefined) {
-          return (closedClaimStatusMap[event.Event])
-        } else {
-          return 'Closed'
-        }
-      } else {
-        return 'Closed'
-      }
+      return (event && event !== null && event !== undefined) ? closedClaimStatusMap[event.Event] : 'Closed'
```
Comment (psoleckimoj) [inline app/services/data/get-closed-claim-status.js:23]: Will delete this, just left the file for now so you can see the simplification of the original call.

File: app/services/data/get-claim-list-for-advanced-search.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const rulesEnum = require('../../../app/constants/region-rules-enum')
 const dateFormatter = require('../date-formatter')
 const statusFormatter = require('../claim-status-formatter')
 const Promise = require('bluebird').Promise
-const getClosedClaimStatus = require('./get-closed-claim-status')
+const getClosedClaimsStatuses = require('./get-closed-claims-statuses')
```
Comment (psoleckimoj) [inline app/services/data/get-claim-list-for-advanced-search.js:8]: Changed the name as it's covering the status of multiple claims now.

File: app/services/data/get-closed-claim-status.js (hunk 1 above)
Comment (tpmcgowan) [reply app/services/data/get-closed-claim-status.js:23]: Nice!

**Review by tpmcgowan — APPROVED** [review]
LGTM!

**Pull request merged**